examples/django/django_exampleapp/README.md

```diff
@@ -0,0 +1,30 @@
+**Django Sample Application for Pybrake**
+
+**About the application:**
+
+The example application provides three GET endpoints:
+
+- /date - return server date and time. 
+- /locations - returns list of available locations. 
+- /weather/{locationName} - returns the weather details for the locations.
+
+
+**Steps to run the API:**
+
+1. Install the dependencies for the application with command - "pip3 install -r requirements.txt"
```

Comment:
It would be nice to follow the layout defined in https://github.com/airbrake/gobrake/blob/master/examples/gin/README.md

Basically, code/command instructions should be on a separate line 

```
like this
```

Therefore what you have becomes 

```sh
pip3 install -r requirements.txt
```

Reply:
Thanks, updated the file in suggested format.